core/network/impl/peer_manager_impl.hpp

```diff
@@ -41,10 +42,7 @@
 
 namespace kagome::network {
 
-  enum class PeerType {
-    PEER_TYPE_IN = 0,
-    PEER_TYPE_OUT
-  };
+  enum class PeerType { PEER_TYPE_IN = 0, PEER_TYPE_OUT };
```

Comment:
I think IN/OUT would be better, to be terse. Especially it is enum-class and can not be used without name of type

Reply:
That part was just clang-formatted. Did not touch this